services/security/src/app/features/users/handlers/has-attribute.handler.ts

```diff
@@ -49,7 +51,7 @@ export default class HasAttributeHandler implements Handler<HasAttributeCommand>
       throw new NotFoundError(`User with id ${userId} doesn't exist.`);
     }
 
-    const attributesIntersection = user?.getAlreadyExists ? user.getAlreadyExists(attributes) : [];
+    const attributesIntersection = await attributesRepository.isUserAlreadyHaveAttributes({user, attributes});
```

Comment:
We've discussed it with @mkopa when we were talking about hasAccess and I don't think we need this info - we can drop it from response.